### PR TITLE
Initial commit including HF-recommended code revisions

### PR DESCRIPTION
**Pull request by eusip**
My collaborators and I within the Affective Computing team at Telecom Paris would like to re-submit our spoken dialogue dataset for publication.

This is a new pull request relative to the [previously closed request](https://github.com/huggingface/datasets/pull/1712) which was reviewed by @lhoestq.

### CODE REVIEW

File: silicone/silicone.py [hunk 1]
```diff
@@ -0,0 +1,620 @@
...
+                "Dialogue_Act": "Dialogue_Act",
+                "Dialogue_ID": "Dialogue_ID",
+            },
+            label_classes=["commissive", "directive", "inform", "question"],
+            label_column="Dialogue_Act",
+            data_url={
+                "train": os.path.join(_URL, "dyda", "train.csv"),
+                "dev": os.path.join(_URL, "dyda", "dev.csv"),
+                "test": os.path.join(_URL, "dyda", "test.csv"),
+            },
```
Comment (lhoestq) [inline silicone/silicone.py:202]:
```suggestion
            data_url={
                "train": _URL + "/dyda/train.csv",
                "dev": _URL + "/dyda/dev.csv",
                "test": _URL + "/dyda/test.csv"),
            },
```
Here we can avoid using `os.path.join` on urls. Indeed on Windows `os.path.join` groups the components using backslashes and the result may not be a valid url.

You can concatenate strings using `+` instead.

Can you also fix that for the other `data_url` dic defined for the other configuration ?

**Review by lhoestq — COMMENTED** [review]
Thanks for the resubmission ! And for the changes in the the tags

I left a few comments.

Also no need to git rebase since it can mess up the github diff ;)

**Comment by eusip** [comment]
Thanks for the feedback. All your comments have been addressed!

**Review by lhoestq — APPROVED** [review]
Thank you !

Looks all good now :)

**Comment by eusip** [comment]
Thank you for your constructive feedback! I now know how to best format future datasets that our team plans to publish in the near future :)

**Comment by lhoestq** [comment]
Awesome ! Looking forward to it :)

**Comment by eusip** [comment]
Hi @lhoestq ! One last question. Our research team would like to distribution a link to this dataset amongst the spoken dialogue research community but the dataset does not show in the dropdown menu at huggingface.co. Is there anything else we must do in order to find the dataset there ?

Once the dataset does show in the dropdown menu, how can I affiliate it with the Telecom Paris organization that I already created at the website ?

**Comment by lhoestq** [comment]
The files are not located in the right place in the repo. Let me move them

**Comment by lhoestq** [comment]
I created a PR at https://github.com/huggingface/datasets/pull/1794

**Comment by lhoestq** [comment]
I just merged the change @eusip, now the dataset page is available at the url:
https://huggingface.co/datasets/silicone

**Comment by eusip** [comment]
Thank you for moving the folder for me :)

**Pull request merged**